fe/fe-core/src/main/java/com/starrocks/sql/ast/AddFieldClause.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.alter.AlterOpType;
+import com.starrocks.sql.ast.StructFieldDesc;
+import com.starrocks.sql.parser.NodePosition;
+
+import java.util.Map;
+
+// clause which is used to add one field to
+public class AddFieldClause extends AlterTableColumnClause {
+    private final String colName;
+    private final StructFieldDesc fieldDesc;
+
+    public String getColName() {
+        return colName;
+    }
+
+    public StructFieldDesc getFieldDesc() {
+        return fieldDesc;
+    }
+
+    public AddFieldClause(String colName, StructFieldDesc fieldDesc, String rollupName, Map<String, String> properties) {
```

Comment:
rollupName is useless, struct fields in rollup should be consistent with base table.